core/transaction_pool.go

```diff
@@ -356,11 +356,12 @@ func (self *TxPool) RemoveTransactions(txs types.Transactions) {
 	self.mu.Lock()
 	defer self.mu.Unlock()
 	for _, tx := range txs {
-		self.removeTx(tx.Hash())
+		self.RemoveTx(tx.Hash())
 	}
 }
 
-func (pool *TxPool) removeTx(hash common.Hash) {
+// Remove transaction from the pool by its hash
+func (pool *TxPool) RemoveTx(hash common.Hash) {
```

Comment:
Note: conventional form for documentation format is
```go
// RemoveTx removes the transaction with the given hash from the pool.
```

(starts with function name, full sentence).